packages/rancher-logging/generated-changes/overlay/templates/loggings/root/logging.yaml

```diff
@@ -22,6 +22,28 @@ spec:
                   name: fluent-bit
         tls:
           enabled: {{ .Values.nodeAgents.tls.enabled | default false }}
+    {{- if .Values.additionalLoggingSources.rke.enabled }}
+    - name: win-agent-rke
+      profile: windows
+      nodeAgentFluentbit:
+        filterKubernetes:
+          Kube_Tag_Prefix: "kuberentes.C.var.lib.rancher.rke.log."
+        inputTail:
+          Path: "C:\\var\\lib\\rancher\\rke\\log"
+        extraVolumeMounts:
+        - source: "/var/lib/rancher/rke/log"
+          destination: "/var/lib/rancher/rke/log"
```

Comment:
To be clear these are the standard logs, not the ones we wrote to get at systemd.

Reply:
They're the kubelet logs and related, but I want to ensure we do not overcollect, so what do you mean exactly?